src/assets/VueOptionVsCompositionAPI/option-vs-composition-api.md

```diff
@@ -0,0 +1,186 @@
+    }
+
+    function decreaseAge() {
+      data.age--
+    }
+   ```
+   The above function changes the `age` of the data defined in the `data` which is reactive.
+
+3. ### Defining computed
+   To define the computed property vue provides the method called `computed` through which we can explicitly define the computed function.
+    ```js
+   //computed
+    const getVotingStatus = computed(() => {
+      return (data.age >= 18) ? 'Eligible' : 'Not Eligible'
+    })
+   ```
+   Here the operation `getVotingStatus` returns the voting status based on the value of the data age.
+
+4. ### Defining watch
+    Similarly, vue has provided a `watch` function to watch the change in any data properties defined.
+    ```js
+   //watcher
+    watch(mainData.age, (oldAge, newAge) => {
+      if(oldAge > newAge) {
+        data.bornYear--
+      } else {
+        data.bornYear++
+      }
+    })
+   ```
+   We have watched data `age` in such a way that when age is increased the data `bornYear` will decrease and vice-versa.
+
+The overall script snippet code looks like this at last for the composition API:
+```js
+import { reactive, toRefs, computed , watch} from "vue";
+export default {
+  setup() {
+    // data
+    const data = reactive({
+      name: 'Sagar Gurung',
+      age: 26,
+      votingStatus: "Not Eligible",
+      bornYear: 1997
+    })
+    const mainData = toRefs(data)
+    // methods
+    function increaseAge() {
+      data.age++
+    }
+    function decreaseAge() {
+      data.age--
+    }
+    //computed
+    const getVotingStatus = computed(() => {
+      return (data.age >= 18) ? 'Eligible' : 'Not Eligible'
+    })
+    //watcher
+    watch(mainData.age, (oldAge, newAge) => {
+      if(oldAge > newAge) {
+        data.bornYear--
+      } else {
+        data.bornYear++
+      }
+    })
+
+    return {
+      ...mainData,
+      increaseAge,
+      decreaseAge,
+      getVotingStatus
+    }
+  }
+}
+```
+In the overall code, we have imported `reactive`, `toRefs`, `computed`, and `watch` needed for building components with Composition API. The `toRefs` is used to convert the data created with `reactive` to plain objects. Also, in Composition API we need to define all those `data`, `computed`, `watch`, and `methods` inside of it as done above.
+At last, all the things that have been defined `data`, `computed`, and `methods` needs to be returned to be exposed to the template.
+
+In this way, we can simply convert a component defined with `Option API` to `Composition API`. But this is just a basic example. There are lots of things that we can perform using composition API.
+I hope this blog gave you some insights on `Composition API` and also how you can replace your component defined in `Option API` with it. But before finishing the blog I would like to highlight some.
+points regarding both `Option API` and `Composition API` so that you can relate which one to prefer before building up your projects.
+
+## Option API and Composition API
```

Comment:
```suggestion
## `Option API` and `Composition API`
```

Reply:
I think this is your conclusion. Maybe choose a title for this section that indicates clearer that this is your conclusion / summary...